### ISSUE TITLE
7.0 beta testing

### ISSUE THREAD

**Issue by nirea** [OpenCollarTeam/OpenCollar#45]
We're getting to release an official OpenCollar 7.0.  The changes are described at https://github.com/OpenCollarTeam/OpenCollar/blob/master/RELEASE_NOTES.md.

We need your help to test!  Please grab a copy of the beta from the OpenCollar R&D group notices in Second Life and try some of the tests described at https://github.com/OpenCollarTeam/OpenCollar/wiki/Manual-Test-Suite.  Then report your results here, whether successs or failure.  Thank you!

**Comment by nirea** [comment]
Awesome, thank you @xxReyes !

There have been no changes to the label code in 7.0 that I'm aware of.   Does the same thing happen on older 6.x releases?  Assuming it does, I think you're probably right about just needing to document it.

**Comment by nirea** [comment]
I want to get this one fixed in the next beta https://github.com/OpenCollarTeam/OpenCollar/issues/55

**Comment by IalaAndromeda** [comment]
Friend tried beta 5 on several 3.997 collars. In every case it hangs at 75%, seemingly just after (or during) moving the animations.
She also tried some 3.995 collars, those worked fine, so it seems a problem highly specific to 3.997 only.

**Comment by nirea** [comment]
@IalaAndromeda I think this was caused by a larger-than-normal number of animations in the object.  When oc_com announced them to the animator prim, it meant a flood of link messages going out, and another flood in response.  But then it would start calling llGiveInventory to move them, and that sleeps the script, so events would start to get dropped if the queue was full.

I've refactored the animation inventory sync code to do an asynchronous loop with no bursts of link messages, and passed you an updater with this code inworld.  In my tests, it's able to move large numbers of anims without issues.
